src/test/java/org/galatea/starter/entrypoint/IexRestControllerTest.java

```diff
@@ -81,4 +81,44 @@ public void testGetLastTradedPriceEmpty() throws Exception {
         .andExpect(jsonPath("$", is(Collections.emptyList())))
         .andReturn();
   }
+
+  @Test
+  public void testGetHistoricalPrices() throws Exception {
```

Comment:
Yes, that is weird. I've tested calling the IEX api directly using their example in the [docs ](https://iexcloud.io/docs/api/#historical-prices) 
(https://cloud.iexapis.com/stable/stock/msft/chart/5d?token=YOUR_TOKEN) and it still returns only 3 days. I've played with the query params and there doesn't seem to be a clear way to get 5 days of data returned.

Reply:
There's a reason for that -- look at the data you're getting back and try and figure out how the API is interpreting the range query and how that affects the data you get.

As far as the code goes, I'd change it to be 3d so that this question wouldn't be raised by someone reading the code. Additionally, once you figure out the reason it's only returning 3 days of data when you query 5 days, that could also be an interesting test case to add.

Reply:
looks like it isn't returning data for days the market is closed, such as weekends. That explains it, thanks! I'll change the tests